packages/core/useShare/index.ts

```diff
@@ -0,0 +1,28 @@
+interface NavigatorWithShare extends Navigator {
+  share?: (data: ShareOptions) => Promise<void>
+  canShare?: (data: ShareOptions) => boolean
+}
+
+type ShareOptions = {
+  title?: string
+  files?: File[]
+  text?: string
+  url?: string
+}
+
+const _navigator = (window.navigator as NavigatorWithShare)
+
+export async function useShare<T>(shareOpts: ShareOptions) {
```

Comment:
Look like the `<T>` is not used anywhere. Can you remove it?

Reply:
I forgot that hehe thanks